tesseract_collision/bullet/src/bullet_cast_simple_manager.cpp

```diff
@@ -134,9 +134,7 @@ bool BulletCastSimpleManager::removeCollisionObject(const std::string& name)
   auto it = link2cow_.find(name);
   if (it != link2cow_.end())
   {
-    cows_.erase(std::find_if(cows_.begin(), cows_.end(), [&name](const auto& p) {
-      return p->getName() == name;
-    }));
```

Comment:
It is late, I am not seeing difference between the two. Can you point out the issue?

Reply:
This commit is just for the clang-format. The full PR changes that from a pointer comparison to a string lambda comparison.